api/wasm/cpp/proxy_wasm_externs.h

```diff
@@ -77,6 +77,22 @@ extern "C" uint32_t proxy_getMetadataStruct(MetadataType type, const char* name_
 extern "C" uint32_t proxy_setMetadataStruct(MetadataType type, const char* name_ptr, size_t name_size,
                                             const char* value_ptr, size_t value_size);
 
+// Generic selectors
+extern "C" uint32_t proxy_resolveSelector0(const char* arg0_ptr, size_t arg0_size,
```

Comment:
Could we find some more specific name than "resolve".  Say "resolveExpression" or "resolveCEL" ?

Reply:
By selector, I was referring to a selector expression. I'm fine with any name, expression is also fine.

Reply:
Since this is the low level ABI I am OK making it more verbose. Let's pick some general suffix for everything of this sort (presuming that at some point we can handle more general expressions and/or setting), say Expression.   Then these would be proxy_getSelectorExpression ?  Then setSelectorExpression, getExpression (which takes general expression string?).  We don't need the other names but I think we should have a plan for if/when we need them. WDYT?

Reply:
Sure, getSelectorExpression sounds good to me.